### DISCUSSION TITLE
Detect Command key press on macOS

### DISCUSSION THREAD

**Discussion by pierrec** [hajimehoshi/ebiten#2540] [Q&A]
The question is in the title :)

**Comment by hajimehoshi** [comment]
Use `KeyMetaLeft`, `KeyMetaRight` and/or `KeyMeta`.

**Comment by pierrec** [comment]
Do you mind if I send a PR with a comment about this in the IsKeyPressed function doc?

**Reply by hajimehoshi** [reply]
Hm, I don' think we need a comment about this. You can try the key names with examples/keyboard https://ebitengine.org/en/examples/keyboard.html

**Reply by pierrec** [reply]
Yes that's what I did but the CMD key is (obviously) never highlighted. Thanks for the answer, works like a charm ofc.
